app/views/labels/_label_li.html.erb

```diff
@@ -35,4 +31,4 @@
       <% end %>
     </li>
   </ul>
-</li>
```

Comment:
Make sure that p contains variable `params[:q]` etc as well. If I search for 'test' it should still search for 'test', but also filter the label.